state-chain/traits/src/mocks/asset_converter.rs

```diff
@@ -33,6 +33,16 @@ impl AssetConverter for MockAssetConverter {
 	) -> Option<(Amount, Amount)> {
 		let input_asset = input_asset.into();
 		let output_asset = output_asset.into();
+		if input_asset == output_asset {
```

Comment:
maybe just a little note to say this snippet is copied from the implementation in the pools pallet